footylib/footylib.py

```diff
@@ -18,13 +18,14 @@
 
 
 class Footy(object):
+    headers = {'User-Agent': 'Mozilla/5.0'}
```

Comment:
There is no reason for this to be a class variable. This makes it even more visible that having it on a self attribute. Having this as a simple variable in init would suffice.